src/standalone/Vegetation/pfts.jl

```diff
@@ -0,0 +1,491 @@
+"""
+This file contains the PFT struct used to prescribe vegetation parameters to the 
+model when running using PFTs, as well as the default PFT scheme. ClimaLand 
+supports running using default PFTs, user defined PFTs, or fully prescribed 
+vegetation parameters.
+
+The default PFTs used in the model use parameter values drawn from defferent 
+sources in the literature. The sources are listed here with abbreviations that
+are used in the comments in the code below.
+
+    Chen, J. M., G. Mo, J. Pisek, J. Liu, F. Deng, M. Ishizawa, 
+    and D. Chan (2012), Effects of foliage clumping on the estimation of global
+    terrestrial gross primary productivity, Global Biogeochem. Cycles, 26, 
+    GB1019, doi:10.1029/2010GB003996.
+    abb. as Chen et al. 2012
+
+    CLM5.0 Technical Note
+    abb. as CLM5.0
+
+    CLM2 PFT Data
+    https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/lnd/clm2/pftdata/
+    abb. as CLM2 PFT Data
+"""
+
+export Pft, default_pfts, params_from_pfts
+
+using DataFrames
+
+# List of parameters that PFTs need to define in order to be valid
+param_list = [
+    # TwoStreamModel parameters
+    :Ω,          # Clumping Index
+    :α_PAR_leaf, # PAR Leaf Reflectance
+    :α_NIR_leaf, # Near-infrared Leaf Reflectance
+    :τ_PAR_leaf, # PAR Leaf Transmittance
+    :τ_NIR_leaf, # Near-infrared Leaf Transmittance
```

Comment:
do we include emissivity?

Reply:
or is that constant over all PFTs

Reply:
Also the leaf angle distribution l_d

Reply:
Have a look at this paper: https://link.springer.com/article/10.1007/s00382-019-04924-9 (Table 1) <img width="394" alt="Screenshot 2024-02-21 at 2 50 00 PM" src="https://github.com/CliMA/ClimaLand.jl/assets/23744195/e955a5ff-55ed-408f-a204-550c77e7e74f">

Reply:
Also, we talked about implementing the leaf angular distribution later on, instead of using 0.5. Can we quickly add what CLM does? It is pretty straightforward. See Eq. 3.3 (https://www2.cesm.ucar.edu/models/cesm2/land/CLM50_Tech_Note.pdf).

𝐺 (𝜇) = 𝜑1 + 𝜑2𝜇 (3.3)

where 𝜑1 = 0.5 − 0.633𝜒𝐿 − 0.33𝜒𝐿2
 and 𝜑2 = 0.877 (1 − 2𝜑1) for −0.4 ≤ 𝜒𝐿 ≤ 0.6. 

𝜒𝐿 is a PFT parameter.

Reply:
I think I should add this change to `ld` in a short follow up PR... Since the formula for `ld` requires the inverse optical depth and the solar zenith angle it will need to be computed within the radiative transfer model itself. So here it this PR I will include XL as a PFT param, and then I will create a second PR that adds this change to ld everywhere.